plugins/module_utils/ec2.py

```diff
@@ -86,7 +86,7 @@ def _botocore_exception_maybe():
     botocore.exceptions instead of assigning from it directly.
     """
     if HAS_BOTO3:
-        return botocore.exceptions.ClientError
+        return (botocore.exceptions.ClientError, botocore.exceptions.WaiterError,)
```

Comment:
@jill / @s-hertel : How do you feel about this ?

I'm thinking it might also be nice to add module.get_waiter with the same tricks as get_client(retry_decorator=...)

Reply:
@tremble It seems like it could cause problems. WaiterError.last_response is not guaranteed to have an error code, and  some waiter errors shouldn't really be retried. If waiters are running into generic throttling errors, should the delay be modified instead? Waiters can also be modified to retry on certain error codes as part of the waiting process. I'm still thinking about this though.